src/Security/PasswordValidator.php

```diff
@@ -76,8 +76,7 @@ class PasswordValidator
     protected $historicalPasswordCount = null;
 
     /**
-     * @deprecated 4.5.0
-     * Minimum password length
+     * @deprecated 4.5.0 Minimum password length
```

Comment:
```suggestion
     * Minimum password length
     * @deprecated 4.5.0 Use setMinLength() instead.
```
Update the message in the notice below as well.

Reply:
Updated